cagent.go

```diff
@@ -34,6 +34,8 @@ type Cagent struct {
 	vmstatLazyInit sync.Once
 	vmWatchers     map[string]types.Provider
 	hwInventory    sync.Once
+	smartInit      sync.Once
```

Comment:
I suggest to change `smart` => `SMART` as done in config.go

Reply:
Resolved?

Reply:
Or initSMART should work as well

Reply:
I would prefer `initSMART` then

Reply:
done